### DISCUSSION TITLE
Cannot change the number of fields of class 'sdpvar' without first typing 'clear classes'.

### DISCUSSION THREAD

**Discussion by Hu-Hust** [yalmip/YALMIP#1422] [Problems using YALMIP]
when I define a 'sdpvar', there is an error: Cannot change the number of fields of class 'sdpvar' without first typing 'clear classes'.
matlab version R2022b & R2024a
yalmip version R20230609

![fig](https://github.com/yalmip/YALMIP/assets/102709492/8b3346f5-1219-46d1-a1e5-888063b83186)

**Comment by johanlofberg** [comment]
Have you done what it tells you to do?

**Reply by Hu-Hust** [reply]
Thanks, I have fixed it.
